src/components/NewClusterModal.js

```diff
@@ -405,6 +482,26 @@ export const NewClusterModal = withModalDrawer({ width: 675 })(class NewClusterM
           h(ButtonPrimary, { onClick: () => this.createCluster() }, ['REPLACE'])
         ])
       ])],
+      ['update', () => h(Fragment, [
+        this.isStopRequired() ?
+          p([
+            'Changing the machine type (increasing or decreasing the # of CPUs or Mem) results in an update that requires a ',
+            b(['restart']),
+            ' of your runtime. This may take a few minutes.  Would you like to proceed? ',
+            b(['(You will not lose any files.)'])
+          ]) :
+          p([
+            'Increasing the disk size or changing the number of workers (when the number of workers is >2) results in a real-time update to your runtime. ',
```

Comment:
I added a `hasChanges` function that I've tested to disable the button as you suggest, but I would like to do it as a separate PR because otherwise identifying the changes after the previous PR approval would be unwieldy

Reply:
that seems reasonable - I had a few other non-blocking comments but can hold those for a future PR as well. I would like to do a bit more testing but with the power out I don't have the current laptop power so will get back to that ASAP